enterprise/frontend/src/metabase-enterprise/embedding/components/InteractiveEmbeddingSettings.tsx

```diff
@@ -36,25 +33,20 @@ export function InteractiveEmbeddingSettings() {
 
   return (
     <SettingsPageWrapper title={t`Interactive embedding`}>
-      <SettingsSection>
-        <EmbeddingToggle
-          settingKey="enable-embedding-interactive"
-          label={t`Enable interactive embedding`}
-        />
-
-        <Box>
-          <SettingHeader id="get-started" title={t`Get started`} />
-          <Button
-            mt="xs"
-            variant="outline"
-            component={ExternalLink}
-            href={quickStartUrl}
-          >{t`Check out the Quickstart`}</Button>
-        </Box>
+      <EmbeddingSettingsCard
+        title={t`Enable interactive embedding`}
+        titleProps={{ fz: "lg" }}
+        // eslint-disable-next-line no-literal-metabase-strings -- This string only shows for admins.
+        description={t`Embed the full power of Metabase into your application to build a custom analytics experience and programmatically manage dashboards and data.`}
+        settingKey="enable-embedding-interactive"
+        links={[{ icon: "bolt", title: t`Quick start`, href: quickStartUrl }]}
+      />
 
+      <SettingsSection>
         <AdminSettingInput
```

Comment:
The description doesn't take all the space, unlike in the design. This happens to the static embedding settings page as well.

<img width="891" height="528" alt="Image" src="https://github.com/user-attachments/assets/0d25ff44-bbf6-4156-bfdc-fe5401e6df80" />